code.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "name": "drb-inland-salinity-ml",
+    "organization": "U.S. Geological Survey",
+    "description": "Code repo for Delaware River Basin machine learning models that predict inland salinity.",
+    "version": "main",
+    "status": "Development",
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://raw.githubusercontent.com/USGS-R/drb-inland-salinity-ml/main/LICENSE.md"
+        }
+      ]
+    },
+    "homepageURL": "https://github.com/USGS-R/drb-inland-salinity-ml/",
+    "downloadURL": "https://github.com/USGS-R/drb-inland-salinity-ml/archive/refs/heads/main.zip",
+    "disclaimerURL": "https://raw.githubusercontent.com/USGS-R/drb-inland-salinity-ml/main/DISCLAIMER.md",
+    "repositoryURL": "https://github.com/USGS-R/drb-inland-salinity-ml.git",
+    "vcs": "git",
+    "laborHours": -1,
```

Comment:
I don't know how this metadata field is used, but should we add some ballpark here (or delete it)?

Reply:
-1 is good. Don't delete it! Most folks leave it at the default. According to Eric Martinez, the main use case for actually using this would be if you had contracted with someone for a given number of hours. Since most of us don't have useful or accurate information to use to populate this, it uses -1 as an NA.

Reply:
Thanks for the context, I promise I won't delete it! 😸

Reply:
If we can come up with a good estimate, yes. I was not tracking programming hours, but we could probably get close with the hours we were collectively assigned to PUMP